src/IssueViz.UnitTests/Editor/QuickActions/QuickFixes/QuickFixSuggestedActionTests.cs

```diff
@@ -154,5 +175,28 @@ private static Mock<ITextSnapshot> CreateTextSnapshot()
             snapShot.SetupGet(ss => ss.Length).Returns(int.MaxValue);
             return snapShot;
         }
+
+        private static QuickFixSuggestedAction CreateTestSubject(IQuickFixVisualization quickFixViz,
```

Comment:
Created a helper method that creates a dummy spanTranslator if one was not given. It returns whatever span it was given, so your tests did not have to change.